### ISSUE TITLE
DatePicker, TimePicker Keyboard Navigation does not work properly in Chrome Desktop 

### ISSUE THREAD

**Issue by dashkan** [telerik/kendo-angular#1945]
## I'm submitting a...

* Bug report 

## Current behavior

Keyboard navigation does not scroll DatePicker and TimePicker in Chrome browser when there is content below the picker. Cannot reproduce this bug in Safari or FireFox.

## Expected behavior

DatePicker, TimePicker popup window should display without closing or scrolling in chrome.

## Minimal reproduction of the problem with instructions
[stackblitz demo](https://stackblitz.com/edit/angular-1ebkse)
**_Resize window so that content scrolls_**. Use keyboard navigation to open datepicker / timepicker dropdown. In chrome the datepicker/timepicker will flash and close. 

[Kendo DatePicker Keyboard Navigation](https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/keyboard-navigation/)
Use keyboard navigation to popup datepicker. The content scrolls to the bottom fo the page and opens the popup.

[Kendo TimePicker Keyboard Navigation Sample](https://www.telerik.com/kendo-angular-ui/components/dateinputs/timepicker/keyboard-navigation/)
Use keyboard navigation to popup timepicker. The content scrolls to the bottom fo the page and opens the popup.

## Environment

Package versions:
- N/A. Can reproduce via Kendo Angular Samples page

Browser:

- Chrome (desktop) version 70.0.3538.102 

System:
- N/A. Can reproduce via Kendo Angular Samples page

**Comment by danielkaradachki** [comment]
@dashkan 
Could you check if this is still an issue? I cannot seem to be able to replicate it. I created a [screen cast](https://www.screencast.com/t/jXZGjSgn7kxb) with the behavior on my side. Let me know if I am missing something.

**Comment by danielkaradachki** [comment]
Closing due to inactivity. Will reopen once sufficient information about the issue is available.
